article/app/views/fragments/inArticleAudio.scala.html

```diff
@@ -0,0 +1,59 @@
+
+<div class="inline-audio_container">
+    <div class="inline-audio_icon">
+    @fragments.inlineSvg("football-weekly-300", "journalism/in-article-audio-player")
+    </div>
+    <div class="inline-audio_content">
+        <div class="inline-audio_content_header">
+            <h3>
+                <span class="kicker">Listen / </span>
+                Class, schoolings and some harsh lessons - Football Weekly
+            </h3>
+        </div>
+        <div class="inline-audio_content_player">
+            <audio
+            class="inline-audio-player-element"
+            src="https://flex.acast.com/audio.guim.co.uk/2018/08/20-54476-gnl.fw.20180820.sj.fw2008.mp3"
+            preload="metadata"
+            data-media-id="gu-audio-5b7467c1e4b016cd4306acc7"
+            data-title="Class, schoolings and some harsh lessons – Football Weekly"
+            data-component="in-article audio"
+            >
+                <p>
+                    Sorry your browser does not support audio - but you can download here
+                    and listen https://flex.acast.com/audio.guim.co.uk/2018/08/20-54476-gnl.fw.20180820.sj.fw2008.mp3
+                </p>
+            </audio>
+            <div class="inline-audio_controls">
+                <div class="inline-audio_button">
+                @fragments.inlineSvg("btn-play", "journalism/in-article-audio-player")
+                </div>
+                <div class="inline-audio_time">
+                    <span id="inline-audio_time-played">00:00</span>
+                    <span> / </span>
+                    <span id="inline-audio_duration"> 1:19:44</span>
```

Comment:
I forgot how far we got with acast but is this value representative of the audio _with_ ads in?

Reply:
Ah - good point, it is overwritten by javascript on load, here: https://github.com/guardian/frontend/pull/20261/files#diff-e9afcc45bb21c1902995678690d8a0c1R67
But i might as well take it out because it won't be seen anyway.